lib/broccoli/angular2-app.js

```diff
@@ -87,10 +87,16 @@ Angular2App.prototype.toTree = function () {
     destDir: 'vendor'
   });
 
+  var publicDirTree = new Funnel(path.join(sourceDir, 'public'), {
```

Comment:
Just to clarify, you'd prefer the `public` directory to live in the project root, then be moved to `dist` from there, yes?